server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/SearchAction.java

```diff
@@ -43,47 +57,60 @@ public void define(WebService.NewController controller) {
       .setDescription("Search for projects associated (or not) to a quality gate.<br/>" +
         "Only authorized projects for current user will be returned.")
       .setSince("4.3")
-      .setResponseExample(Resources.getResource(this.getClass(), "example-search.json"))
+      .setResponseExample(Resources.getResource(this.getClass(), "search-example.json"))
       .setHandler(this);
 
-    action.createParam(QualityGatesWsParameters.PARAM_GATE_ID)
+    action.createParam(PARAM_GATE_ID)
       .setDescription("Quality Gate ID")
       .setRequired(true)
       .setExampleValue("1");
 
-    action.createParam(QualityGatesWsParameters.PARAM_QUERY)
+    action.createParam(PARAM_QUERY)
       .setDescription("To search for projects containing this string. If this parameter is set, \"selected\" is set to \"all\".")
       .setExampleValue("abc");
 
     action.addSelectionModeParam();
 
-    action.createParam(QualityGatesWsParameters.PARAM_PAGE)
+    action.createParam(PARAM_PAGE)
       .setDescription("Page number")
       .setDefaultValue("1")
       .setExampleValue("2");
 
-    action.createParam(QualityGatesWsParameters.PARAM_PAGE_SIZE)
+    action.createParam(PARAM_PAGE_SIZE)
       .setDescription("Page size")
       .setExampleValue("10");
+
+    wsSupport.createOrganizationParam(action);
   }
 
   @Override
   public void handle(Request request, Response response) {
-    QgateProjectFinder.Association associations = projectFinder.find(ProjectQgateAssociationQuery.builder()
-      .gateId(request.mandatoryParam(QualityGatesWsParameters.PARAM_GATE_ID))
-      .membership(request.param(QualityGatesWsParameters.PARAM_QUERY) == null ? request.param(Param.SELECTED) : ProjectQgateAssociationQuery.ANY)
-      .projectSearch(request.param(QualityGatesWsParameters.PARAM_QUERY))
-      .pageIndex(request.paramAsInt(QualityGatesWsParameters.PARAM_PAGE))
-      .pageSize(request.paramAsInt(QualityGatesWsParameters.PARAM_PAGE_SIZE))
-      .build());
-    try (JsonWriter writer = response.newJsonWriter()) {
-      writer.beginObject().prop("more", associations.hasMoreResults());
-      writer.name("results").beginArray();
+
+    try (DbSession dbSession = dbClient.openSession(false)) {
+
+      OrganizationDto organization = wsSupport.getOrganization(dbSession, request);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "organization" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1481)